<launch><group ns="leader"><node name="turtle3" pkg="turtlesim" respawn="true" type="turtlesim_node" /><node name="teleop" pkg="turtlesim" required="true" type="turtle_teleop_key" /></group><node name="turtle1" pkg="turtlesim" respawn="true" type="turtlesim_node" /><node name="turtle2" pkg="turtlesim" respawn="true" type="turtlesim_node" /><node name="turtle4" pkg="turtlesim" respawn="true" type="turtlesim_node" /><param name="leaderAgentNameSpace" type="str" value="leader" /><node name="publishLeaderPosToLeaderPosHolder" output="screen" pkg="suivit" type="publishLeaderPosToLeaderPosHolder.py" /><node name="publishLeaderPosToFollowers" output="screen" pkg="suivit" type="publishLeaderPosToFollowers.py" /><node name="customizeFrames" pkg="suivit" type="customizeFrames.py" /></launch>